packages/dds/tree/src/test/cursorTestSuite.ts

```diff
@@ -566,13 +568,93 @@ function testTreeCursor<TData, TCursor extends ITreeCursor>(config: {
                             assert.equal(cursor.getFieldLength(), 1);
                             cursor.enterNode(0);
                         });
+
+                        it(`traversal with key: ${key.toString()}`, () => {
+                            const dataWithKey = dataFactory();
+                            const cursor = cursorFactory(dataWithKey);
+                            checkTraversal(cursor, parent);
+                        });
                     }
                 }
             });
+
+            it("traverse with no keys", () => {
+                const data = withLocalKeys([]);
+                const cursor = cursorFactory(data);
+                checkTraversal(cursor, parent);
+            });
+
+            describe("cursor prefix tests", () => {
+                it("at root", () => {
+                    const data = withLocalKeys([]);
+                    const cursor = cursorFactory(data);
+                    expectEqualPaths(cursor.getPath(), parent);
+                    expectEqualPaths(cursor.getPath(undefined), parent);
+                    expectEqualPaths(cursor.getPath({}), parent);
+
+                    const prefixParent: UpPath = {
+                        parent: undefined,
+                        parentField: brand("prefixParentField"),
+                        parentIndex: 5,
+                    };
+                    if (extraRoot) {
+                        // TODO
```

Comment:
I need to fix this test coverage gap, and see if this is why BasiChunkCursor is passing tests (its missing the prefix logic, and should fail)